devtools/server/actors/replay/auth.js

```diff
@@ -289,6 +304,11 @@ function openSigninPage() {
           await new Promise(resolve => setTimeout(resolve, 3000));
         } else {
           setReplayRefreshToken(resp.data.closeAuthRequest.token);
+
+          // refresh the token immediately to exchange it for an access token
+          // and acquire a new refresh token
+          await refresh();
```

Comment:
This is really just moved from `setReplayRefreshToken` to here so we're not invoking side effects directly from a setter.